packages/api/src/controllers/access-control.ts

```diff
@@ -135,6 +135,13 @@ app.post(
       }
     }
 
+    if (req.body.pub && playbackPolicyType !== "public") {
+      if (req.body.pub === req.config.accessControlAdminPublicKey) {
+        res.status(204);
+        return res.end();
+      }
+    }
```

Comment:
could have it in a single check
```suggestion
    if (playbackPolicyType !== "public" && req.body.pub === req.config.accessControlAdminPublicKey) {
        res.status(204);
        return res.end();
    }
```